src/Stratis.Bitcoin.Features.Consensus/ConsensusLoop.cs

```diff
@@ -201,9 +238,9 @@ public class ConsensusLoop
 
             this.asyncLoop = this.asyncLoopFactory.Run($"Consensus Loop", async (token) =>
             {
-                await this.PullerLoopAsync(this.nodeLifetime.ApplicationStopping).ConfigureAwait(false);
-            },
-            this.nodeLifetime.ApplicationStopping,
+                await this.PullerLoopAsync().ConfigureAwait(false);
+            }, 
+            this.nodeLifetime.ApplicationStopping, 
```

Comment:
everyone else is removing these end line spaces ...

Reply:
You mean put it in one line?